system/Assertion.cfc

```diff
@@ -1454,16 +1454,13 @@ component {
 	 * @throws InvalidTargetType - If we can normalize it to a struct
 	 */
 	private function normalizeToStruct( any target ){
-		if ( isStruct( arguments.target ) ) {
-			return arguments.target;
-		}
 		if ( isQuery( arguments.target ) ) {
```

Comment:
I see your request, but then this introduces exceptions to be thrown in other locations when the normalization can't happen for things like NON structs.